test/Test/Fission/Web/Client/Error.hs

```diff
@@ -0,0 +1,53 @@
+module Test.Fission.Web.Client.Error (tests) where
+
+import           Test.Fission.Prelude as Mock
+
+
+import Fission.Web.Client.Error
+import Network.HTTP.Types.Status
+import           Fission.Internal.Fixture            as Fixture
+
+tests :: IO TestTree
+tests = do
+
+  -----------------------
+  -- EFFECTFUL SESSION --
+  -----------------------
+
+  Mock.Session 
+    { effectLog = _effectLog :: [OpenUnion '[LogMsg]]
+    , result = validResult 
+    } <- runMock defaultConfig do
+        checkStatus [status502, status504] (Right (0 :: Integer))
+
+  Mock.Session 
+    { effectLog = _effectLog :: [OpenUnion '[LogMsg]]
+    , result = errResult 
+    } <- runMock defaultConfig do
+        checkStatus [status502, status504] (Left Fixture.failure502)
+
+  Mock.Session 
+    { effectLog = _effectLog :: [OpenUnion '[LogMsg]]
```

Comment:
If you're tossing the `_effectLog`, you may not need to draw the values (unless the compiler complains)

```haskell
Mock.Session {result = errResult } <- runMock defaultConfig do
  checkStatus [status502, status504] (Left Fixture.failure502)
```

Reply:
nope unfortunately the compiler complains
```
    • You require open sum type to contain the following element:
          LogMsg
      However, given list can store elements only of the following types:
          effs0
```